src/trackinfocache.cpp

```diff
@@ -0,0 +1,378 @@
+#include "trackinfocache.h"
```

Comment:
If you're up for it, it would be nice to name this TrackCache and move it to src/track/.

Once the open PRs that touch TIO are in, I plan to rename TrackInfoObject to Track and move it into src/track/track.h/cpp.